r55/src/exec.rs

```diff
@@ -70,15 +71,15 @@ struct RVEmu {
 
 fn riscv_context(frame: &Frame) -> Option<RVEmu> {
     let interpreter = frame.interpreter();
-    if matches!(interpreter.bytecode.get(0), Some(0xFF)) {
-        let emu = setup_from_elf(&interpreter.bytecode[1..], &interpreter.contract.input);
-        Some(RVEmu {
-            emu,
-            returned_data_destiny: None,
-        })
-    } else {
-        None
-    }
+
+    let Some((0xFF, bytecode)) = interpreter.bytecode.split_first() else {
+        return None;
+    };
+    let emu = setup_from_elf(bytecode, &interpreter.contract.input);
+    Some(RVEmu {
+        emu,
+        returned_data_destiny: None,
+    })
```

Comment:
Personally OK with picking `0xFF` as the prefix and calling it -- @leonardoalt does it make sense to make an EOF version for this where we "keep it simple" containers-wise, just so we can reserve the prefix for RiscV?

Reply:
Yea could do that too. We'd need to wrap into an EOF container at compile time, I guess it should be fine? Feels weird to wrap an ELF binary inside EOF though, so maybe we should also do some translation ELF -> EOF at some point.